boards/arm/stm32g071_disco/doc/index.rst

```diff
@@ -0,0 +1,182 @@
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| MPU       | on-chip    | arm memory protection unit          |
++-----------+------------+-------------------------------------+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| CLOCK     | on-chip    | reset and clock control             |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | flash memory                        |
++-----------+------------+-------------------------------------+
+| COUNTER   | on-chip    | rtc                                 |
++-----------+------------+-------------------------------------+
+| WATCHDOG  | on-chip    | independent watchdog                |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+| ADC       | on-chip    | adc                                 |
++-----------+------------+-------------------------------------+
+| DAC       | on-chip    | dac                                 |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported in this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+``boards/arm/stm32g071_disco/stm32g071_disco_defconfig``
+
+Connections and IOs
+===================
+
+Each of the GPIO pins can be configured by software as output (push-pull or open-drain), as
+input (with or without pull-up or pull-down), or as peripheral alternate function. Most of the
+GPIO pins are shared with digital or analog alternate functions. All GPIOs are high current
+capable except for analog inputs.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+- UART_3 TX/RX       : PC10/PC11 (ST-Link Virtual Port Com)
+- Button (JOY_SEL)   : PC0
+- Button (JOY_LEFT)  : PC1
+- Button (JOY_DOWN)  : PC2
+- Button (JOY_RIGHT) : PC3
+- Button (JOY_UP)    : PC4
+- LED (TO_REC)       : PD9
+- LED (TO_PLUG)      : PD8
+- LED (SINK_SPY)     : PD6
```

Comment:
- LED (SINK_SPY)     : PD**5**

Reply:
I wasn't sure if the B was needed. Thank you for the clarification.